commands/roll.py

```diff
@@ -0,0 +1,5 @@
+async def roll(ctx):
+    """ Rolls a pseudorandom number from 0 to 100 """
+    num = randint(0, 100)
+    num = str(num)
```

Comment:
These two lines can probably be combined:
```suggestion
    num = str(randint(0, 100))
```